pkg/cmd/pulumi/stack_rm.go

```diff
@@ -75,7 +75,10 @@ func newStackRmCmd() *cobra.Command {
 			if err != nil {
 				if hasResources {
 					return result.Errorf(
-						"'%s' still has resources; removal rejected; pass --force to override", s.Ref())
+						"'%s' still has resources; removal rejected. Possible actions:\n"+
+							"- Make sure the correct stack is selected\n"+
```

Comment:
for this line, can we state the name of the stack they currently have selected?

Reply:
Thank you! I added the stack name and reworded. Not sure I love the result - do you have a suggestion to improve further?